lib/app/http_request.nit

```diff
@@ -126,7 +126,7 @@ end
 #
 # Prints on communication errors and when the server returns an HTTP status code not in the 200s.
 #
-# ~~~
+# ~~~nitish
```

Comment:
Well, it was the only failing test in this PR. Do you want me to copy the commit into a distinct PR?